schedule/containers/sle_image_on_google_gke.yaml

```diff
@@ -1,7 +1,8 @@
 ---
-name: sle_image_on_amazon_gke
+name: sle_image_on_google_gke
```

Comment:
Please get rid of this file in favour of `lib/main_containers.pm` :smile:

Reply:
Let's do that in another PR, as we have all the K8S schedules in `schedule/containers`, ok?

Reply:
Yes yes :smile: